plan/sort.go

```diff
@@ -80,11 +80,27 @@ func Sort(p *blip.Plan) []SortedLevel {
 						Metrics: []string{},
 					}
 				}
-				higherDomain.Metrics = append(higherDomain.Metrics, lower.Collect[domain].Metrics...)
+
+				seen := toSet(higherDomain.Metrics)
```

Comment:
When adding new, important blocks of logic, also add a code comment to explain what/why. Future engineers will thank you.